src/mono/wasm/runtime/library_mono.js

```diff
@@ -386,6 +386,10 @@ var MonoSupportLib = {
 
 			for (var i in props) {
 				var p = props [i];
+
+				if (p.name == null)
```

Comment:
This looks like a symptom of some other bug. Also, this function gets completely reworked in https://github.com/dotnet/runtime/pull/41480/files#diff-e04dc79ff3f1f7b84d42424ece6ba562 , which hopefully, will be merged soon. I can try to reproduce this issue on top of that PR.